lib/schema.js

```diff
@@ -208,20 +203,18 @@ export default class Schema {
         });
         indexName = indexName.join('_and_');
       }
-
-      this.indexes[indexName] = {path: prop, name: indexName, indexType: indexType, compound: true};
+      compound = true;
     } else {
-      if (!this.descriptor[prop]){
-        throw new Error('Descriptor not found for this field');
-      }
-
-      this.descriptor[prop].index = true;
-      this.descriptor[prop] = _.extend(this.descriptor[prop], options);
+      if (!indexName) {
+        indexName = prop
+        if (trailingDigitReg.test(indexName)) {
+          indexName = indexName[indexName.length - 2];
```

Comment:
Just copying what's happening here, but looking at it, doesn't seem to make much sense.
https://github.com/atrites/lounge/blob/master/lib/schema.utils.js#L92